spec/pascals_triangle_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'pascals_triangle'
+
+describe PascalsTriangle do
+
+  describe '.triangle' do
```

Comment:
@StefanGajic do you have a triangle method? This should say `to_s` because that's the method you are testing in this `describe` block.